msm/msm

```diff
@@ -23,114 +23,181 @@
 # This script assists in the installation and management of
 # skills loaded from Github.  
 
-set -e
+mycroft_skill_folder=${mycroft_skill_folder:-"/opt/mycroft/skills"}
+if [[ ! -d "${mycroft_skill_folder}" ]] ; then
+    echo "Unable to access ${mycroft_skill_folder}!"
+    exit 101
+fi 
+
+# picroft?
+if [[ "$(hostname)" == 'picroft' ]] || [[ -x /home/pi/bin/cli ]] ; then
+  picroft='true'
+else
+  picroft='false'
+  if [[ -r /etc/bash_completion.d/virtualenvwrapper ]]; then
+    source /etc/bash_completion.d/virtualenvwrapper
+  else
+    if locate virtualenvwrapper ; then
+      if ! source $(locate virtualenvwrapper) ; then
+        echo "Unable to locate virtualenvwrapper.sh, will not be able to install skills!"
+        vwrap='false'
+      fi
+    fi
+  fi
+fi
 
-mycroft_skill_folder="/opt/mycroft/skills"
-mycroft_virtualenv=~/.virtualenvs/mycroft/bin/activate
+default_skills="alarm audio-record configuration date-time desktop-launcher ip joke hello-world media npr-news naptime pairing personal reminder installer singing speak spelling stop stock volume weather wiki wolfram-alpha mark1-demo"
```

Comment:
These should be `skill-...` so the match is exact otherwise the install will fail if multiple skills matches. (Right now media fails on this)